openapi/components/schemas/Cashier/CashierStrategyAmounts.yaml

```
@@ -0,0 +1,47 @@
+type: object
+description: Cashier strategy amounts configuration.
+required:
+  - calculator
+  - baseAmount
+  - increments
+properties:
+  calculator:
+    type: string
+    description: |-
+Cashier amounts calculator.
+This field specifies whether amounts are calculated using an absolute or percentage calculation.
+
+Amounts are specified in `baseAmount`, and increments are specified in `increments`.
+
+Example: `baseAmount: 10` and `increments: [20, 50, 100]`.
+
+- If the calculator is set to `absolute`: the amounts displayed to the customer are `[10, 30, 60, 110]`.
+- If the calculator is set to `percent`: the amounts displayed to the customer are `[10, 12, 15, 20]`.
```

I may have added bad indentation here 😬 Sorry! 

```suggestion
    description: |-
        Cashier amounts calculator.
        This field specifies whether amounts are calculated using an absolute or percentage calculation.
        
        Amounts are specified in `baseAmount`, and increments are specified in `increments`.
        
        Example: `baseAmount: 10` and `increments: [20, 50, 100]`.
        
        - If the calculator is set to `absolute`: the amounts displayed to the customer are `[10, 30, 60, 110]`.
        - If the calculator is set to `percent`: the amounts displayed to the customer are `[10, 12, 15, 20]`.
```